src/Search.php

```diff
@@ -1156,7 +1156,16 @@ public static function explodeWithID($pattern, $subject)
      */
     public static function joinDropdownTranslations($alias, $table, $itemtype, $field)
     {
-        return SQLProvider::joinDropdownTranslations($alias, $table, $itemtype, $field);
+        /** @var \DBmysql $DB */
+        global $DB;
+
+        Toolbox::deprecated();
```

Comment:
An entry in the changelog should be added.

Reply:
Done